open-company/open-company-web
# Finances component

card: https://trello.com/c/CfOeMlz4

test:
- run `lein test-all!`
- load the sample data in the api project, instructions [here](https://github.com/open-company/open-company-api/pull/13)
- check that all the data are loaded as expected
- try to edit the title of each section
- try to edit the body of each section
- try to edit the finances report data
- navigate the history of the sections
- merge if you can
